std/math.d

```diff
@@ -668,37 +628,6 @@ deprecated
     assert(50 - abs(-100) == -50);
 }
 
-version (TestComplex)
-deprecated
-@safe pure nothrow @nogc unittest
-{
-    import std.meta : AliasSeq;
-    static foreach (T; AliasSeq!(cfloat, cdouble, creal))
-    {{
-        T f = -12+3i;
-        assert(abs(f) == hypot(f.re, f.im));
-        assert(abs(-f) == hypot(f.re, f.im));
-    }}
-}
-
-import std.meta : AliasSeq;
```

Comment:
Why was this removed ? It's a breaking change.

Reply:
I'm having flashbacks here – didn't we fix selective imports to be private a few years ago?

Reply:
(i.e. the infamous "bug 314")

Reply:
As @dnadlinger said, did not anticipate there being a `public:` being in effect here.

Reply:
https://issues.dlang.org/show_bug.cgi?id=20836